src/Core/CHANGELOG.md

```diff
@@ -2,9 +2,40 @@
 
 ## 0.5.0
 
+### Added
+
+- Internal `AsyncAws\Core\Response` to encapsulate the HTTP client.
```

Comment:
should add or remove final dot for consistency

Reply:
Sorry. Thank you